backend/usage/models.py

```diff
@@ -26,7 +26,18 @@ class Usage(BaseModel):
     run_id = models.CharField(
         max_length=255, null=True, blank=True, db_comment="Identifier for the run"
     )
-    usage_type = models.CharField(max_length=255, db_comment="Type of usage")
+    usage_type = models.CharField(
```

Comment:
Can we make it enum choices? not necessory.

Reply:
Done